mongodb_atlas/metadata.csv

```diff
@@ -95,3 +95,15 @@ mongodb.atlas.system.network.bytes.out,rate,60,byte,second,The average rate of p
 mongodb.atlas.network.bytes.in,rate,60,byte,second,The average rate of physical (after any wire compression) bytes sent to this database server per second,0,mongodb_atlas,network bytes in,
 mongodb.atlas.network.bytes.out,rate,60,byte,second,The average rate of physical (after any wire compression) bytes sent from this database server per second,0,mongodb_atlas,network bytes out,
 mongodb.atlas.network.num.requests,rate,60,request,second,The average rate of requests sent to this database server per second,0,mongodb_atlas,network num requests,
+mongodb.atlas.search.index.stats.commands.getmore,rate,60,operation,,Number of getmore commands run on all Atlas Search queries,0,mongodb_atlas,num search index getmore command,
```

Comment:
I put 60 second interval these metric because `Datadog measures each metric once per minute` in https://www.mongodb.com/docs/atlas/tutorial/datadog-integration/#performance-metrics-available-to-datadog